packages/realm/binding/apple/platform.mm

```diff
@@ -18,6 +18,7 @@
 
 #include "../platform.hpp"
 
+#include <memory>
```

Comment:
Is this strictly needed?

Reply:
I don't think so, it's probably just a leftover